tasks/install_shell.sh

```diff
@@ -237,7 +237,6 @@ if [ -f "$PT__installdir/facts/tasks/bash.sh" ]; then
       "10.12") platform_version="10.12";;
       "10.13") platform_version="10.13";;
       "10.14") platform_version="10.14";;
```

Comment:
Could you remove these older versions as well the way we did in install-puppet?

Reply:
Thanks @mhashizume,

Removed all the 10.x lines and removed the if condition since it will always be true now after the removal.